PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditRequest.kt

```diff
@@ -12,7 +12,8 @@ import com.braintreepayments.api.core.ExperimentalBetaApi
 @ExperimentalBetaApi
 open class PayPalVaultEditRequest
 @JvmOverloads constructor(
-    open val editPayPalVaultId: String
+    open val editPayPalVaultId: String,
+    open val hasUserLocationConsent: Boolean
```

Comment:
Also doc strings

Reply:
And we should probably merge the testing PR first since this should require test changes too